src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(health) {
```

Comment:
Constructor method of `Animal` class should take `name` parameter as well, and set the corresponding property